### ISSUE TITLE
Wait-HPOVTaskComplete gives error on output since last update

### ISSUE THREAD

**Issue by VincentBerger** [HewlettPackard/POSH-HPOneView#484]
### Expected Behavior


### Actual Behavior
With the last Library update where you did some work on Wait-HPOVTaskComplete I'm getting a non-terminating error. This is the output I'm getting
`
PS /Users/vincent> New-HPOVServerProfile -Name "Cluster Node 3" -ServerProfileTemplate $SPT -server $server                                                                                                                                                                                                                                                                                        sort: invalid option -- P                                                                                                                                       Usage: sort [-bcCdfigMmnrsuz] [-kPOS1[,POS2] ... ] [+POS1 [-POS2]] [-S memsize] [-T tmpdir] [-t separator] [-o outfile] [--batch-size size] [--files0-from file] [--heapsort] [--mergesort] [--radixsort] [--qsort] [--mmap] [--parallel thread_no] [--human-numeric-sort] [--version-sort] [--random-sort [--random-source file]] [--compress-program program] [file ...]                                                                                                                                                                                                                                                                                       Create - Cluster Node 3                                                                                                                                            
Running                                                                                                                                                         [ooooooooooooooooooooooooooooo                                                                                                                       ]         Assign profile - Synergy-Encl-3, bay 11                                                                                                                            
Running                                                                                                                                                         [                                                                                                                                                  ]         Create - BootVol-Gen10                                                                                                                                             
Running                                                                                                                                                         [                                                                                                                                                  ]         Configure connections -                                                                                                                                            
Running                                                                                                                                                         [                                                                                                                                                  ]         Configure SAN storage -                                                                                                                                            
Running
`
Progress bars do not render well in the issue editor but point is the invalid option on sort at the beginning. I believe it comes from this line:
`                                        if ($CurrentChildOperation = ($_childTaskObj.progressUpdates | sort -Property timeStamp -Descending | Select -First 1).statusUpdate)
`
You use 'sort' assuming it is an alias for Sort-Item I guess but on my system it isn't and there actually is a sort shell command which is apparently what's being called here. I guess it would be similar on a Linux system


### Version Information
`ApplianceVersion: 5.20.00.419867.00 5.20.2452.2750 /Users/vincent/.local/share/powershell/Modules/HPOneView.520/5.20.2452.2750
`
Output from `$PSVersionTable` on your Windows Host:

```PowerShell
Name                           Value
----                           -----
PSVersion                      7.0.0
PSEdition                      Core
GitCommitId                    7.0.0
OS                             Darwin 19.5.0 Darwin Kernel Version 19.5.0: Tue May 26 20:41:44 PDT 2020; root:xnu-6153.121.2~2/RELEASE_X86…
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

```

**Comment by ChrisLynchHPE** [comment]
It is supposed to be `Sort-Object`.  To work around this, you can do the following:

```powershell
PS /mnt/c/Users/chris> New-Alias sort Sort-Object
PS /mnt/c/Users/chris> get-help sort

NAME
    Sort-Object

SYNTAX
    Sort-Object [[-Property] <Object[]>] [-Stable] [-Descending] [-Unique] [-InputObject <psobject>] [-Culture <string>
    ] [-CaseSensitive] [<CommonParameters>]

    Sort-Object [[-Property] <Object[]>] -Top <int> [-Descending] [-Unique] [-InputObject <psobject>] [-Culture <string
    >] [-CaseSensitive] [<CommonParameters>]

    Sort-Object [[-Property] <Object[]>] -Bottom <int> [-Descending] [-Unique] [-InputObject <psobject>] [-Culture <str
    ing>] [-CaseSensitive] [<CommonParameters>]


ALIASES
    sort


REMARKS
    Get-Help cannot find the Help files for this cmdlet on this computer. It is displaying only partial help.
        -- To download and install Help files for the module that includes this cmdlet, use Update-Help.
        -- To view the Help topic for this cmdlet online, type: "Get-Help Sort-Object -Online" or
           go to https://go.microsoft.com/fwlink/?LinkID=2097038.

```

**Comment by ChrisLynchHPE** [comment]
This will be fixed in the next library update.

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 5.20.2470.2147](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v5.20.2470.2147) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.520/5.20.2470.2147).

Please verify before this issue is closed.

**Comment by VincentBerger** [comment]
Verified OK. Closing
